templates/taxi/car_confirm_delete.html

```diff
@@ -1,10 +1,11 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Delete car?</h1>
+  <h3>Are you sure, you want to delete car "{{ car.model }}"?</h3>
   <form action="" method="post">
     {% csrf_token %}
 
     <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{% url 'taxi:driver-list' %}" class="btn btn-secondary">Cancel</a>
```

Comment:
According to the checklist, a `Cancel` button should lead to the previous page the user was on. Here it leads to the driver list page instead. Consider using JavaScript or Django's messages framework to implement this.